azure/azure_style.yaml

```diff
@@ -44,17 +44,11 @@ jobs:
   pool:
     vmImage: 'ubuntu-20.04'
   continueOnError: ${{ parameters.IGNORE_STYLE }}
-  strategy:
-    matrix:
-      'python37':
-        PYTHON_VERSION: '3.7'
-      'python38':
-        PYTHON_VERSION: '3.8'
   steps:
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: $(PYTHON_VERSION)
+      versionSpec: 3.8
```

Comment:
In a few other spots we have this as:

```
  - task: UsePythonVersion@0
    inputs:
      versionSpec: '3.8'
```

with `` around 3.8. I am not sure if we want that for consistency here too.

Reply:
Oh yeah, I am not sure whether those are needed but I'll add them in.

Reply:
Got it — sounds good. Thanks for fixing this!